### PR TITLE
Misc queue from 2018 (sorry, lost sources)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1919,6 +1919,7 @@ awefully->awfully
 awnser->answer
 awya->away
 axises->axes
+ayway->anyway
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest away too.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2102,6 +2103,7 @@ beloging->belonging
 belogs->belongs
 belond->belong
 beloning->belonging
+belove->below
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest beloved.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -2192,6 +2194,7 @@ Blitzkreig->Blitzkrieg
 bload->bloat
 bloaded->bloated
 bloccks->blocks
+blocs->blocks
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the singular

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -2394,6 +2397,7 @@ buss->bus
 busses->buses
 bussiness->business
 butons->buttons
+buts->but
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest butts

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Valid in the US? https://dictionary.cambridge.org/dictionary/english/buts

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Also "no ifs no buts"

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -2602,6 +2607,7 @@ carreid->carried
 carrers->careers
 Carribbean->Caribbean
 Carribean->Caribbean
+carridge->carriage
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest cartridge

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -7738,6 +7747,7 @@ fonetic->phonetic
 fontain->fountain, contain,
 fontains->fountains, contains,
 fontier->frontier
+fontonfig->fontconfig
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Specific

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed. But - as I say - this is "codespell". So it should make sense to leave it.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Fair point, see what @larsoner thinks. Personally I think it should probably be in a separate dictionary.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, I think after my rambling and given the Cambridge hit this should probably be removed @Gelma .

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: No problem. We can discuss about it in the future.

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: I'm fine with keeping the more coding-specific ones so long as they are "sufficiently common". `fontconfig` is pretty common I think

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Do you want to add this back in then @Gelma ?

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Ok, thanks.

**Pull request merged**